batch/batch/worker/worker.py

```diff
@@ -1193,9 +1193,12 @@ async def container_config(self):
                 'resources': {
                     "devices": [{"allow": False, "access": "rwm"}],
                     'cpu': {'shares': weight},
-                    'memory': {
-                        'limit': self.memory_in_bytes,
-                        'reservation': self.memory_in_bytes,
+                    'unified': {  # https://github.com/opencontainers/runtime-spec/blob/main/config-linux.md
+                        'memory.max': str(self.memory_in_bytes),
+                        'memory.low': '0',
+                        'memory.high': str(int(0.95 * self.memory_in_bytes)),
+                        'memory.min': str(self.memory_in_bytes),
```

Comment:
While I think setting high to 95% will help ease off the OOM situation, we need to set max (and by extension min) to something less than 100% of the RAM available on the machine, otherwise the worker process can be starved of RAM.